src/components/navigation/top-nav/top-nav.tsx

```diff
@@ -53,7 +105,9 @@ const TopNav: React.FC<TopNavProps> = ({ navItems }) => {
           ariaHasPopup={true}
           size={ButtonSize.small}
           ariaControls="main"
-          onClick={handleMenuClick}></IconButton>
+          onClick={handleMenuClick}
```

Comment:
On line 102, the label should not contain the word "button" as it is repetitive. arial-label="Menu" will be perfect!